lib/davinci_dtr_test_kit/client_groups/dinner_static/dtr_full_ehr_questionnaire_workflow_group.rb

```diff
@@ -6,6 +6,8 @@
 require_relative 'dtr_full_ehr_rendering_enabled_questions_attestation_test'
 require_relative 'dtr_full_ehr_store_attestation_test'
 require_relative 'dtr_full_ehr_prepopulation_representation_attestation_test'
```

Comment:
this file is removed - remove this line

Reply:
https://github.com/inferno-framework/davinci-dtr-test-kit/pull/19/commits/4f414d7eeb0f70edf51fe9bb784f9f1f4338d132